### PR TITLE
Document project resources section of contribution guide 

### PR DESCRIPTION
**Pull request by maayanf24**
Continue the work on the closed PR: https://github.com/submariner-io/submariner-website/pull/723

### CODE REVIEW

File: src/content/development/contribution-guide/_index.en.md [hunk 1]
```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub Projects] to manage releases. Read [Tracking Progress on your Project Board] to learn more on how to work with
```
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Submariner uses [GitHub Projects] to manage releases. Read [Tracking Progress on your Project Board] to learn more about how to work with
```

File: src/content/development/contribution-guide/_index.en.md [hunk 2]
```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
...
+projects.
 
 ### Backlog Board
 
-To be expanded
+[The backlog board] hosts issues, bugs and features which are not a part of a particular release board. While each issue is opened in
+its corresponding repository, this board gives an aggregated view of all open issues across all repositories. The board has total of seven
+columns. Any issue or epic opened should be assigned to `Backlog` Projects in `Backlog` column. Epics which are candidates to be included in
```
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
columns. Any new issues or epics should typically be assigned to the `Backlog` column. Epics which are candidates to be included in
```

File: src/content/development/contribution-guide/_index.en.md [hunk 3]
```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
...
+the next release are moved to the `Next Version Candidate` column. During [planning](#planning), all epics from the `Next
+Version Candidate` column are getting review and triage. Epics which are selected to the next release are moved to
```
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
Version Candidate` column are reviewed and triaged. Epics which are selected to be in the next release are moved to
```

File: src/content/development/contribution-guide/_index.en.md [hunk 4]
```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
...
+[current release board](#current-release-board). The next 3 columns sorts issues by priority. All unassigned issues are sorted into these
```
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
the [current release board](#current-release-board). The next 3 columns sort issues by priority. All unassigned issues are sorted into these
```

File: src/content/development/contribution-guide/_index.en.md [hunk 5]
```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
...
+three columns based on the priority discussed in the team meeting. Issues from these columns are ready to be worked on. Any leftover work
+from the previous release goes to the `Work in Progess column`. `Close?` column holds any issue/epic that could probably be closed for
```
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
from the previous release goes to the `Work in Progress column`. The `Close?` column holds any issues/epics that can probably be closed for
```

File: src/content/development/contribution-guide/_index.en.md [hunk 6]
```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
...
+Current release work is tracked in [the latest board]. The board has 5 columns. The first one, `Schedule and Epics`, hosts all properly
+triaged epics targeted for the current release and the schedule for the current release. Triaged issues are under the `To do` column. When
+an issue is being worked on, it is moved to `In Progress` column. When a PR for the issue is pushed, the issue is moved to `In Review`
+column. The PR has the corresponding issue linked from the GitHub UI and is not tracked on the board. Once the PR is merged, the issue
+is moved to `Done` column.
 
 ### Enhancements Repository
 
-To be expanded
+The [Enhancements repository] in used for tracking [epics](#epics) as issues, and their corresponding enhancement proposals (the epics design)
```
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
The [Enhancements repository] is used for tracking [epics](#epics) as issues, and their corresponding enhancement proposals (the epics design)
```

File: src/content/development/contribution-guide/_index.en.md [hunk 7]
```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
...
+as [pull requests to the enhancements repository]. Any medium or small enhancement should be opened as an issue to the relevant repository,
```
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
as [pull requests to the enhancements repository]. Any medium or small enhancement should be opened as an issue in the relevant repository,
```

File: src/content/development/contribution-guide/_index.en.md [hunk 8]
```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
...
+or as a general [enhancement Request issue].
```
Comment (dfarrell07) [inline src/content/development/contribution-guide/_index.en.md]:
```suggestion
or as a general [Enhancement Request issue].
```

**Review by dfarrell07 — CHANGES_REQUESTED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Review by Jaanki — APPROVED** [review]

**Review by mkolesnik — APPROVED** [review]

File: src/content/development/contribution-guide/_index.en.md [hunk 9]
```diff
@@ -10,23 +10,39 @@ in order for their contributions to the project to be addressed in a timely mann
...
 
-To be expanded
+[The backlog board] hosts issues, bugs and features which are not a part of a particular release board. While each issue is opened in
+its corresponding repository, this board gives an aggregated view of all open issues across all repositories. The board has total of seven
+columns. Any new issues or epics should typically be assigned to the `Backlog` column. Epics which are candidates to be included in
+the next release are moved to the `Next Version Candidate` column. During [planning](#planning), all epics from the `Next
+Version Candidate` column are reviewed and triaged. Epics which are selected to be in the next release are moved to
+the [current release board](#current-release-board). The next 3 columns sort issues by priority. All unassigned issues are sorted into these
+three columns based on the priority discussed in the team meeting. Issues from these columns are ready to be worked on. Any leftover work
+from the previous release goes to the `Work in Progress column`. The `Close?` column holds any issues/epics that can probably be closed for
```
Comment (sridhargaddam) [inline src/content/development/contribution-guide/_index.en.md:25]: > The `Close?` column...

Are you waiting for any inputs from someone?

Comment (maayanf24) [reply src/content/development/contribution-guide/_index.en.md:25]: I just wanted to give some time to folks to review. I think we can merge now

Comment (sridhargaddam) [reply src/content/development/contribution-guide/_index.en.md:25]: Okay. So do you want to remove the "?"

Comment (maayanf24) [reply src/content/development/contribution-guide/_index.en.md:25]: This is the name of the column on the Backlog board - `Close?`, so it included in the description along side with the other columns description

**Review by skitt — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Pull request merged**